styles/styles.css

```
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 400;
+  src: url("fonts/ptsans-400.woff2") format("woff2");
+  font-display: swap;
+}
+
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 700;
+  src: url("fonts/ptsans-700.woff2") format("woff2");
+  font-display: swap;
+}
+/*
+body {
+  font-family: "PT Sans", sans-serif;
+  font-size: 16px;
+  line-height: 21px;
+  color: #000000;
+  background-color: #FFFFFF;
+}
+*/
+
+.logo-main {
+  background-image: url("images/background.jpg");
```

Пути к картинкам прописываются относительно `css` файла в котором они пишутся (хотя бы потому, что один и тот же `css` файл может подключаться к разным `html`.

Т.е. сначала нужно выйти на папку выше  (`..`)
```suggestion
  background-image: url("../images/background.jpg");
```